### PR TITLE
makuosan: update to 1.3.6

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6.8 Server
Xcode 3.2.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: net/makuosan/Portfile [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-version                 1.2.1
-revision        4
-homepage                http://lab.klab.org/wiki/Makuosan
+version                 1.3.6
+revision                0
+homepage                https://sourceforge.net/projects/makuosan
 categories              net
-platforms               darwin
 maintainers             nomaintainer
-use_parallel_build      yes
```
Comment (reneeotten) [inline net/makuosan/Portfile:14]: `port lint` complains about a missing `license` field; please add that, from upstreams sourceforge site it looks like it's `GPL3` but please verify from the source files.

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: net/makuosan/Portfile (hunk 1 above)
Comment (barracuda156) [reply net/makuosan/Portfile:14]: Thank you for noticing, will fix it.

**Comment by barracuda156** [comment]
@reneeotten License added.

File: net/makuosan/Portfile [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 
 name                    makuosan
-version                 1.2.1
-revision        4
-homepage                http://lab.klab.org/wiki/Makuosan
+version                 1.3.6
+revision                0
+license                 GPL-3
+homepage                https://sourceforge.net/projects/makuosan \
+                        https://github.com/yasui0906/makuosan
```
[the remark this answers is not in the document]
Comment (barracuda156) [reply net/makuosan/Portfile:13]: @reneeotten Yes, it does. I have had such ports, it worked fine.

**Pull request merged**